code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -305,6 +305,18 @@
 		if("record_recipe")
 			recording_recipe = list()
 			. = TRUE
+		if("delete_recipe")
+			var/deletion_target = stripped_input(usr,"Name","please type the recipe you wish to delete", "Recipe", MAX_NAME_LEN)
```

Comment:
Just use `saved_recipes` as a list `input()`...

Reply:
Working on it.

Reply:
How's that?
![image](https://user-images.githubusercontent.com/39484008/219132911-f9ae3928-c651-467d-aa67-4d50fc1d18cf.png)